WSL/enterprise-current.md

```diff
@@ -0,0 +1,50 @@
+---
+title: WSL for Enterprise
+description: Resources and instructions on how to best use the WSL in an Enterprise environment.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, debian, suse, windows 10, enterprise, deployment, offline, packaging, store, distribution, installation, install
+ms.date: 09/04/2018
+ms.topic: article
+ms.assetid: 7afaeacf-435a-4e58-bff0-a9f0d75b8a51
+ms.custom: seodec18
+---
+
+# Windows Subsystem for Linux for Enterprise
+
+## Creating your WSL image
+
+Once WSL is installed on your machine, you should [choose a Linux distribution](https://docs.microsoft.com/windows/wsl/install-win10#step-6---install-your-linux-distribution-of-choice) to run from the Windows store.
+
```

Comment:
Is it okay to assume here that the users reading this will already understand what an "image" is in this context?
We also need more context for this section... this jumps straight into "Once installed", but doesn't actually explain how to create a WSL image or set any context for it... could we add something like...
"The process of capturing the state of all the software and data on a workstation computer for the purpose of copying it onto other computers, or creating a [system image](https://en.wikipedia.org/wiki/System_image) is quite common in enterprise organizations. It helps to create a consistent and well-defined work environment across the company. Including WSL on this system image could be a great addition to your enterprise tool selection, enabling your company to access Linux tools and the distribution of your choosing.
To add WSL to your system image, begin by following the [WSL install steps](https://docs.microsoft.com/windows/wsl/install-win10). *Not sure if this is correct..?* By enabling the optional WSL and Virtual Machine Platform components, including the latest Linux kernal, and setting the default to WSL 2, your image will allow users to jump directly to [choosing a Linux distribution](https://docs.microsoft.com/windows/wsl/install-win10#step-6---install-your-linux-distribution-of-choice) to run from the Microsoft store. *We no longer use the terminology "Windows store".*
I'm not sure this process is correct - but point being that this could benefit from more context setting.